clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -56,10 +57,13 @@
                                                                           new ArrayOf(STRING),
                                                                           "An array of topics to fetch metadata for. If no topics are specified fetch metadata for all topics."));
 
-    public static final Schema BROKER = new Schema(new Field("node_id", INT32, "The broker id."),
+    public static final Schema METADATA_REQUEST_V1 = new Schema(new Field("topics",
+                                                                          new ArrayOf(STRING, true),
```

Comment:
Personally, I'd introduce a `ArrayOf.nullable` static factory method as I think that would be more readable than the `true` here (since we don't have named arguments in Java).